providers/fips/self_test_data.inc

```diff
@@ -386,6 +386,28 @@ static const ST_KAT_PARAM pbkdf2_params[] = {
     ST_KAT_PARAM_END()
 };
 
+/*
+ * Sample parameters from NIST ACVP demo server
+ */
+static const char snmpkdf_digest[] = "SHA1";
```

Comment:
Do they have any sha256 samples? Might be better to use if they did,,,

Reply:
No, the RFC does not support it.